### DISCUSSION TITLE
Track parent ingredients w/ cocktails I can make

### DISCUSSION THREAD

**Discussion by ccozzi13** [karlomikus/bar-assistant#316] [General]
Hi, I just got up and running and enjoying bar-assistant so far.  I was playing around with 'Track parent ingredients as a substitutes' as I built out my shelf - and am I overlooking how this works?  It seems to be pulling back more cocktails than I can really create.

In the simplest case - I just have Gin on my shelf:

![image](https://github.com/user-attachments/assets/58349593-c4d9-45cc-85bd-1b609fa58f5a)

This end's up pulling in, for example, a dry martini:

![image](https://github.com/user-attachments/assets/2ba6859b-9219-41dd-b84c-4b64cf403c8f)

Though it does realize vermouth isn't shelved once drilled in:

![image](https://github.com/user-attachments/assets/7899065f-da3f-48cf-aafa-2875ff1767b4)

It seems to consistently happen when just one ingredient is missing - ex, if I add lemon juice - Saturn pops up:

![image](https://github.com/user-attachments/assets/8e4bd1b6-2010-465b-b32f-0ecb3311b0ed)
![image](https://github.com/user-attachments/assets/1e3283d7-0a92-44f3-96b2-fe8433a81496)
 
------------
Salt Rim: v2.17.0
Bar Assistant: v3.18.0
Meilisearch: v1.6.2

**Comment by karlomikus** [comment]
Thanks for the report, indeed there was an issue.
Should be fixed with latest release. Only thing that's unchanged is the representation on the frontend, the ingredient will be marked with missing even though you have one of the parent ingredients.

This will be fixed with future updates.
